File: utilsLibrary/src/main/java/com/zeropercenthappy/utilslibrary/ZPHTextUtils.java
package com.zeropercenthappy.utilslibrary;

public class ZPHTextUtils {
    public static boolean notEmpty(String... values) {
        boolean result = false;
        for (String value : values) {
            if (android.text.TextUtils.isEmpty(value)) {
                result = true;
                break;
            }
        }
        return result;
    }
}

File: utilsLibrary/src/main/java/com/zeropercenthappy/utilslibrary/utils/ZPHTextUtils.kt
package com.zeropercenthappy.utilslibrary.utils

import android.text.TextUtils

object ZPHTextUtils {
    @JvmStatic
    fun notEmpty(vararg values: String): Boolean {
        for (value in values) {
            if (TextUtils.isEmpty(value)) {
                return false
            }
        }
        return true
    }

    @JvmStatic
    fun hasEmpty(vararg values: String): Boolean {
        for (value in values) {
            if (TextUtils.isEmpty(value)) {
                return true
            }
        }
        return false
    }
}
